torq.q

```diff
@@ -610,14 +611,18 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+.proc.run:{[id;a]
```

Comment:
I prefer `.proc.try` given we are using a try-catch / error trap, but maybe `run` is more explanatory